### ISSUE TITLE
Should we change the trendline?

### ISSUE THREAD

**Issue by LudwikJaniuk** [Klimatbyran/klimatkollen-deprecated#456]
The current trendline is calculated based on an interpolation of the last five years. We present this trendline as extending far into the future. This model has the advantage of being easy to compute and easy to understand. At the same time, it has a few disadvantages:
 - The long-term trend can seem unreasonable such as in the case of Pajala (trending an extremely large increase) or Stockholm (Trending reaching zero in a few years). 
 - It is very sensitive to only changes in the last five years, ignoring more long-term trends. One newly-opened factory can for example completely skew the direction.
 - Users might perceive the trendline as more reliable than it is, i.e. an actual prediction based on data and analysis. Even the best predictions (e.g. done by research institutes and/or consultancy firms) rapidly lose precision after just a few years. What we have isn't even a prediction, it's a naive trend interpolation. 
 
 With that said, it's not immediately obvious what a "solution" would look like. Some options are:
 1. Use a different model for predicting future emissions
 2. Keep the trendline but present it differently, e.g. add an "uncertainty cone", fade it after a few years, or something similar
 3. Do nothing, keep it as-is.

I'm creating this issue to have a place for discussion, and look forward to hearing people's opinions.
 
![image](https://github.com/Klimatbyran/klimatkollen/assets/6298393/6e628a1d-5467-4213-b59b-21ba31256505)
![image](https://github.com/Klimatbyran/klimatkollen/assets/6298393/1b163a63-9196-4ac3-8c9f-bd167a0f7275)

**Comment by sirisab** [comment]
I agree with the description of the problem and I vote for suggestion 2 – to fade it out after a few years. Maybe three years? Or five, since the trendline is calculated from the last five years?

**Comment by Greenheart** [comment]
This is a important question, and we need to involve several stakeholders to resolve it. Right now we don't have the capacity for it, but hopefully in the near future :)
